### PR TITLE
Add notificationservice module

### PR DESCRIPTION
**Pull request by kapildv**
[Here](https://github.com/patilsan90/transact/commit/d86086d529cda75a98d9dedfedb630b5b848ea10) I have added the notification service module.

### CODE REVIEW

File: transact/app/src/main/java/com/algo/transact/notificationservice/NotificationTokenService.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package com.algo.transact.notificationservice;
```
Comment (patilsan90) [inline transact/app/src/main/java/com/algo/transact/notificationservice/NotificationTokenService.java]: Change the package name to notification_service

File: transact/app/src/main/java/com/algo/transact/notificationservice/OnReciveNotificationService.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+import com.google.firebase.messaging.FirebaseMessagingService;
+import com.google.firebase.messaging.RemoteMessage;
+
+/**
+ * If the Application is in fore ground, then will recieve the notification in onMessageRecieved.
+ * Here we can check the type of notification by using appropriate methods.
+ */
+
+public class OnReciveNotificationService extends FirebaseMessagingService {
```
Comment (patilsan90) [inline transact/app/src/main/java/com/algo/transact/notificationservice/OnReciveNotificationService.java]: "Recive" spell change to "Receive"

Comment (kapildv) [reply transact/app/src/main/java/com/algo/transact/notificationservice/OnReciveNotificationService.java]: Done. [Here](https://github.com/patilsan90/transact/pull/4/commits/311a42939ce57ffe2a90f1fb8e0893406b5465c6)

File: transact/app/src/main/java/com/algo/transact/notificationservice/NotificationTokenService.java (hunk 1 above)
Comment (kapildv) [reply transact/app/src/main/java/com/algo/transact/notificationservice/NotificationTokenService.java]: Done [Here](https://github.com/patilsan90/transact/pull/4/commits/311a42939ce57ffe2a90f1fb8e0893406b5465c6)

**Pull request merged**